core/common/pom.xml

```diff
@@ -131,7 +131,7 @@
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-tcnative-boringssl-static</artifactId>
-      <version>2.0.26.Final</version>
+      <version>2.0.34.Final</version>
```

Comment:
i tried this version and it worked. can you double check on your laptop?

Reply:
Hmm doesn't seem to work for me
```Caused by: java.lang.NoClassDefFoundError: io/netty/internal/tcnative/AsyncSSLPrivateKeyMethod
	at io.netty.handler.ssl.SslContext.newClientContextInternal(SslContext.java:832)
	at io.netty.handler.ssl.SslContextBuilder.build(SslContextBuilder.java:611)```